docs/source/en/model_doc/rwkv.mdx

```diff
@@ -64,43 +64,43 @@ torch.allclose(torch.cat([output_one, output_two], dim=1), output_whole, atol=1e
 
 ## Rwkv attention and the recurrent formulas
 
-In a traditional auto-regressive Transformer, attention is written as
+In a traditional autoregressive Transformer, attention is written as
 
 $$O = \hbox{softmax}(QK^{T} / \sqrt{d}) V$$
 
-with \\(Q\\), \\(K\\) and \\(V\\) are matrices of shape `seq_len x hidden_size` named query, key and value (they are actually bigger matrices with a batch dimension and an attention head dimension but we're only interested in the last two, which is where the matrix product is taken, so for the sake of simplicity we only consider those two). The product \\(QK^{T}\\) then has shape `seq_len x seq_len` and we can take the maxtrix product with \\(V\\) to get the output \\(O\\) of the same shape as the others.  
+where \\(Q\\), \\(K\\) and \\(V\\) are matrices of shape `seq_len x hidden_size` named the Query, Key, and Value (in reality, these are matrices with higher dimensionality to account for a batch dimension and an attention head dimension. In this instance, we are only interested in the last two dimensions, which is where the matrix product is taken, so for the sake of simplicity we only consider those two). The product \\(QK^{T}\\) then has shape `seq_len x seq_len` and we can take the maxtrix product with \\(V\\) to get the output \\(O\\) of the same shape as the others.  
```

Comment:
* The previous paragraph is easier to understand. We can still cut a few words from that though :) 
* I realise `query` etc are names we're giving, but they're written with lowercases everywhere in the library, so let's keep to that pattern 
* There's a missing noun after `Value` -> I can't say `they are the blue` (blue what?)

```suggestion
where \\(Q\\), \\(K\\) and \\(V\\) are matrices of shape `seq_len x hidden_size` named query, key and value (they are actually bigger matrices with a batch dimension and an attention head dimension but we're only interested in the last two, where the matrix product is taken, so for the sake of simplicity we only consider these). The product \\(QK^{T}\\) then has shape `seq_len x seq_len` and we can take the maxtrix product with \\(V\\) to get the output \\(O\\) of the same shape as the others.
```